### ISSUE TITLE
Support multi-page switchlists in general.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bowdidge/switchlist#9]
```
Some switchlists (SP-style and B-7) support multi-page.  That code should be 
generalized so any multi-page form style switchlist can be printed.
```

---

Original issue reported on code.google.com by `rwbowdi...@gmail.com` on 24 Apr 2011 at 5:27

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed/improved in the next release - probably 0.7.1.
```

---

Original comment by `rwbowdi...@gmail.com` on 21 Jun 2011 at 7:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.7.2.  Mostly.
```

---

Original comment by `rwbowdi...@gmail.com` on 22 Jun 2011 at 4:59
- Changed state: **Fixed**
